### ISSUE TITLE
axis.title.x and axis.title.y

### ISSUE THREAD

**Issue by calligross** [calligross/ggthemeassist#16]
axis.title.x and axis.title.y should be added, but all values must be NULL until changed by the user.

**Comment by ottlngr** [comment]
Could you specify this? axis.title.x and axis.title.y are supported... Or do you mean axis.text, axis.ticks and axis.line for x & y separately?

**Comment by calligross** [comment]
Yes, I meant separately. But I guess you are right, it might be too many input widgets. I think we should declare this to a feature for a future release. I have some more ideas which need a complete overhaul of different functions.
